qpc/scan/utils.py

```diff
@@ -95,12 +95,41 @@ def _get_optional_products(disabled_optional_products):
     return optional_product_status
```

Comment:
Is this set correctly? I was only looking as for the edit case if you handle empty array to mean "unset" everything to defaults you may need to make a change here.  Feel like this is the reverse though.

Reply:
`optional_product_status` is kinda a weird name. I know its from a previous PR, but it was one of the huge ones.